### PR TITLE
Disable pam_console_password whent is not set

### PR DESCRIPTION
**Pull request by jarekherisz**
#### Pull Request (PR) description
In bareos, if the pam_console_password parameter is defined in the configuration file, bareos tries to use the PAM method.

This results in the inability to log into the webui with an error:
"Sorry, cannot authenticate. Wrong username, password or SSL / TLS handshake failed."

In the logs we can see:
"AUTH: 'admin' is not a defined PAM console on DIR, referer: / bareos-webui / auth / login"

#### This Pull Request (PR) fixes the following issues
- If $pam_console_password is undef it is no transferred in to"/etc/bareos-webui/directors.ini"
- Added repository release key for version 21

### CODE REVIEW

File: manifests/repository.pp [hunk 1]
```diff
@@ -42,6 +42,9 @@
 
   if $gpg_key_fingerprint {
     $_gpg_key_fingerprint = $gpg_key_fingerprint
+  } elsif versioncmp($release, '21') >= 0 {
+    # >= bareos 20
+    $_gpg_key_fingerprint = '91DA 1DC3 564A E20A 76C4  CA88 E019 57D6 C9FE D482'
```
Comment (root-expert) [inline manifests/repository.pp:47]: This should be in a different PR

**Review by root-expert — CHANGES_REQUESTED** [review]
Thanks for the PR! Take a look at the inline comment please :smile:

File: manifests/repository.pp (hunk 1 above)
Comment (jarekherisz) [reply manifests/repository.pp]: I had to create a new PR to link it to the right brunch: https://github.com/voxpupuli/puppet-bareos/pull/99
Now there should be order in these changes.
